src/test/java/picard/sam/markduplicates/UmiAwareMarkDuplicatesWithMateCigarTest.java

```diff
@@ -175,25 +208,31 @@ public void testBadUmis(List<String> umis, List<String> assignedUmi, final List<
         // unique
         double effectiveLength3_1_1 = -(3./5.)*Math.log(3./5.)/Math.log(4.) -2*(1./5.)*Math.log(1./5.)/Math.log(4.);
 
+        double effectiveLength_N = -(3./4.)*Math.log(3./4.)/Math.log(4.) -(1./4.)*Math.log(1./4.)/Math.log(4.);
+
         // estimatedBaseQualityk_n is the phred scaled base quality score where k of n bases are incorrect
         double estimatedBaseQuality1_20 = QualityUtil.getPhredScoreFromErrorProbability(1./20.);
         double estimatedBaseQuality3_20 = QualityUtil.getPhredScoreFromErrorProbability(3./20.);
+        double estimatedBaseQuality_N = QualityUtil.getPhredScoreFromErrorProbability(2./16.);
+
+        double estimatedPercentWithN = 3./7.;
```

Comment:
Could you change this variable to have 3 and 7 in the name?
estimatedPercentWithN3_7?

Reply:
done